### PR TITLE
Upgrade to symfony 3

### PR DESCRIPTION
**Pull request by lepiaf**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,8 +12,8 @@
         }
     ],
     "require": {
-        "php": ">=5.3.3",
-        "symfony/framework-bundle" : "~2.1",
+        "php": ">=5.5.9",
```
Comment (ch3ric) [inline composer.json]: nope

File: composer.json [hunk 2]
```diff
@@ -12,8 +12,8 @@
...
+        "symfony/framework-bundle" : "~3.1",
```
Comment (ch3ric) [inline composer.json]: >=2.1

**Comment by ch3ric** [comment]
create new tag 0.2.5

File: composer.json (hunk 1 above)
Comment (lepiaf) [inline composer.json]: done

File: composer.json (hunk 2 above)
Comment (lepiaf) [inline composer.json]: done

**Pull request merged**